pkg/cgroups/cgroups.go

```diff
@@ -96,9 +99,8 @@ func getCgroupRootForKind() (string, error) {
 		}
 
 		if strings.HasPrefix(line, "0::") { // cgroup v2 starts with "0::"
-			re := regexp.MustCompile(`^.*kubelet`)
```

Comment:
There will be only one instace of cgroup v2, so the regexp will be compiled only once.

Reply:
OK. Sorry I didn't know this before. I will close this pr.